lib/vitalsigns/command.rb

```diff
@@ -2,15 +2,25 @@
 require 'json'
 
 module Vitalsigns
+  # Base class for commands
   class Command
-    def execute(command, options={})
-      Open3.popen3(command) do |stdin, stdout, stderr, wait_thr|
-        return stdout.read
+    def execute(command, _options = {})
```

Comment:
Okay. Good to know thanks @kgaikwad 
I would suggest we can keep arguments for `Open3`, but should remove `_options` from `execute` methods.